x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps.tsx

```diff
@@ -9,15 +9,24 @@ import React, { useCallback, useMemo } from 'react';
 import { EuiText, EuiButton, EuiSpacer } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
+import semver from 'semver';
 
 import type { AgentPolicy, PackagePolicy } from '../../types';
-import { sendGetOneAgentPolicy } from '../../hooks';
+import { sendGetOneAgentPolicy, useKibanaVersion } from '../../hooks';
 import { FLEET_SERVER_PACKAGE } from '../../constants';
 
 import { EnrollmentStepAgentPolicy } from './agent_policy_selection';
 import { AdvancedAgentAuthenticationSettings } from './advanced_agent_authentication_settings';
 
 export const DownloadStep = () => {
+  const kibanaVersion = useKibanaVersion();
+  const kibanaVersionURLString = useMemo(
```

Comment:
It's not a blocker for me, but in the main branch this will go to https://www.elastic.co/downloads/past-releases/elastic-agent-8-0-0 which 404's.

I don't know if it's better to leave the existing https://ela.st/download-elastic-agent for snapshot builds or use this invalid URL

One optional enhancement for another time/PR is to use the results from https://artifacts-api.elastic.co/v1/search/8.0-SNAPSHOT/elastic-agent to make a list of links like below (in the flyout?)
<img width="396" alt="Screen Shot 2021-07-07 at 12 10 52 PM" src="https://user-images.githubusercontent.com/57655/124800500-b648b600-df23-11eb-94bd-8f8584e280a6.png">

None of these are requirements to ship, IMO; just pointing out what I noticed